app/src/pages/PageProjects.re

```diff
@@ -76,16 +97,17 @@ module GetProjectsQuery = ReasonApolloHooks.Query.Make(GetProjectsConfig);
 [@react.component]
 let make = () => {
   let (simple, _full) = GetProjectsQuery.use();
-  let dispatch = Store.useDispatch();
 
   let content =
     switch (simple) {
-    | Error(err) =>
-      StoreMiddleware.Thunk(
-        ThunkAlerts.showAlert(StoreAlerts.Error, err##message),
-      )
-      |> dispatch;
-      React.null;
+    | Error(_error) =>
+      <ErrorMessage>
+        <Text> {React.string("Coundn't load projects.")} </Text>
+        <br />
+        <Text>
+          {React.string("Please make sure the proxy is running.")}
```

Comment:
Maybe this one is better suited as a log message and the end-user sees something like backend not reachable. What you think?

Reply:
Good point! Changed messaging. Also moved the card to a molecule and added the molecule to the styleguide.